langchain/src/base_language/index.ts

```diff
@@ -25,51 +21,14 @@ export abstract class BaseLangChain {
    */
   verbose: boolean;
 
-  callbackManager?: CallbackManager;
-
-  protected async configureCallbackManager(
-    callbackManager?: CallbackManager
-  ): Promise<CallbackManager | undefined> {
-    let callbackManager_;
-    if (callbackManager) {
-      callbackManager_ = callbackManager.copy(
-        this.callbackManager?.handlers,
-        false
-      );
-    } else if (this.callbackManager) {
-      callbackManager_ = new CallbackManager();
-      callbackManager_.setHandlers(this.callbackManager?.handlers ?? [], false);
-    }
-    // eslint-disable-next-line no-process-env
-    if (this.verbose || process.env.LANGCHAIN_TRACING !== undefined) {
-      if (!callbackManager_) {
-        callbackManager_ = new CallbackManager();
-      }
-      const consoleHandler = new ConsoleCallbackHandler();
-      if (
-        this.verbose &&
-        !callbackManager_.handlers.some(
-          (handler) => handler.name === consoleHandler.name
-        )
-      ) {
-        callbackManager_.addHandler(consoleHandler, false);
-      }
-      if (
-        // eslint-disable-next-line no-process-env
-        process.env.LANGCHAIN_TRACING !== undefined &&
-        !callbackManager_.handlers.some(
-          (handler) => handler.name === "langchain_tracer"
-        )
-      ) {
-        callbackManager_.addHandler(await getTracingCallbackHandler(), true);
-      }
-    }
-    return callbackManager_;
-  }
+  callbacks?: CallbackManager | BaseCallbackHandler[];
 
-  constructor(verbose?: boolean, callbackManager?: CallbackManager) {
+  constructor(
+    verbose?: boolean,
```

Comment:
I think this should take an args object with these keys rather than the positional args